src/views/Releases/ListReleases/index.jsx

```diff
@@ -64,12 +67,28 @@ function ListPermissions(props) {
     }
   }, [hash, filteredReleases]);
 
+  const handleAccessChange = ({ release, checked }) => {
+    setDialogState({
+      open: true,
+      title: checked ? 'Read/Write?' : 'Read Only?',
+      confirmText: 'Yes',
+      body: `This would make the permissions of release ${release.name} ${
```

Comment:
Let's avoid using the word `permissions` here, because it overlaps with the idea of user permissions. How about `This would make ${release.name} writable/read only`?